login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreen.tsx

```diff
@@ -0,0 +1,134 @@
+import { CheckCircle } from '@mui/icons-material';
+import { Box } from '@mui/material';
+import React, { useState } from 'react';
+import { Trans } from 'react-i18next';
+import { SimpleDialog } from '../../../components';
+import { LinkStyles } from '../../../styles';
+import { useAuthContext } from '../../contexts';
+import { useLanguageLocale } from '../../hooks';
+import { SuccessScreenBase } from '../SuccessScreen';
+import { ForgotPasswordScreenBase } from './ForgotPasswordScreenBase';
+import { ForgotPasswordScreenProps } from './types';
+
+export const ForgotPasswordScreen: React.FC<ForgotPasswordScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const { actions, navigate, routeConfig } = useAuthContext();
+
+    const [emailInput, setEmailInput] = useState('');
+    const [isLoading, setIsLoading] = useState(false);
+    const [showErrorDialog, setShowErrorDialog] = useState(false);
+
+    const EMAIL_REGEX = /^[A-Z0-9._%+'-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+    const emailValidator = (email: string): boolean | string =>
+        new RegExp(EMAIL_REGEX).test(email) ? true : t('bluiAuth:MESSAGES.EMAIL_ENTRY_ERROR');
+
+    const {
+        WorkflowCardBaseProps: workflowCardBaseProps = {
+            loading: isLoading,
+        },
+        WorkflowCardHeaderProps: workflowCardHeaderProps = {
+            title: t('bluiAuth:HEADER.FORGOT_PASSWORD'),
+        },
+        contactPhone = '',
+        WorkflowCardInstructionProps: workflowCardInstructionProps = {
+            instructions: (
+                <Box>
+                    <Trans i18nKey={'bluiAuth:FORGOT_PASSWORD.INSTRUCTIONS_ALT'} values={{ phone: contactPhone }}>
+                        Please enter the account email associated with the account. If this email has an account with
```

Comment:
we shouldn't need to hardcode the instruction verbiage here, it should pull from translations